docs/overview.md

```diff
@@ -11,62 +11,59 @@ weight=-99
 
 # Docker Swarm overview
```

Comment:
Rolfe, I think this piece needs a major correction.  It is not high-level enough and covers content we have elsewhere.  At this point I may need to take this on to make that correction.  Let's see how the network pro comes out.